lib/app/lift.js

```diff
@@ -46,10 +38,9 @@ module.exports = function lift(configOverride, cb) {
       });
     }
 
+    sails.initialize();
```

Comment:
This is now a synchronous function, we don't need to load it as part of an `async` chain.